stream_alert_cli_module/runner.py

```diff
@@ -604,6 +596,8 @@ def configure_output(options):
     props = output.get_user_defined_properties()
 
     for name, prop in props.iteritems():
+        # TODO: Don't use protected method here
+        # pylint: disable=protected-access
```

Comment:
I would prefer a new copy of the namedtuple rather than accessing an internal method in the namedtuple class. I just wanted to verify that this was expected behavior